src/util.cpp

```diff
@@ -901,7 +901,7 @@ std::string LicenseInfo()
            FormatParagraph(strprintf(_("Copyright (C) 2009-%i The Bitcoin Core Developers"), COPYRIGHT_YEAR)) + "\n" +
            FormatParagraph(strprintf(_("Copyright (C) 2015-%i The Zcash Developers"), COPYRIGHT_YEAR)) + "\n" +
            FormatParagraph(strprintf(_("Copyright (C) 2015-%i zdeveloper.org"), COPYRIGHT_YEAR)) + "\n" +
-           FormatParagraph(strprintf(_("Copyright (C) 2017-%i BitcoinZ team"), COPYRIGHT_YEAR)) + "\n" +
+           FormatParagraph(strprintf(_("Copyright (C) 2017-%i BitcoinZ Core Developers"), COPYRIGHT_YEAR)) + "\n" +
```

Comment:
Leave credit for BitcoinZ team

Reply:
I was trying to be consistent with other areas: https://github.com/btcz/bitcoinz/pull/42/files#diff-b9f4ea3882f3685ed32cfb7b663cdec3R1 -- but I failed to do so.

So should I update everything to:
- BitcoinZ Team
- BitcoinZ Developers
- BitcoinZ Core Developers

I don't care which we choose.

Reply:
🥇